frontend/package.json

```diff
@@ -11,23 +11,32 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
+    "babel-plugin-relay": "^10.1.3",
     "graphql": "^15.5.0",
     "leaflet": "^1.7.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-leaflet": "^3.0.5",
+    "react-relay": "^10.1.3",
     "react-scripts": "4.0.1",
+    "relay-hooks": "^4.1.1",
     "subscriptions-transport-ws": "^0.9.18",
     "typescript": "^4.0.3",
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "yarn prebuild && react-scripts start",
+    "prebuild": "yarn relay",
+    "build": "yarn prebuild && react-scripts build",
+    "test": "yarn prebuild && react-scripts test",
+    "lint": "eslint --max-warnings=0",
+    "lint:all": "yarn lint './src/**/*.{ts,tsx}'",
     "eject": "react-scripts eject",
     "prettier": "prettier --write .",
-    "prettier-check": "prettier --check ."
+    "prettier:check": "prettier --check .",
+    "relay": "relay-compiler",
+    "schema:update": "sh scripts/import-graphql-schema.sh",
```

Comment:
tämä nyt siis npm contextista kutsuu bash scriptin joka taas kutsuu npm komennon. Olisiko helpompi vain tehdä näin:

`"schema:update": "wait_on http://${HASURA}/healthz && graphqurl/bin/run 'http://${HASURA}/v1beta1/relay' --introspect > ../schema.graphql"`
niin et tarvitse mitään bash scriptiä...